src/shared/components/post/post.tsx

```diff
@@ -475,6 +475,26 @@ export class Post extends Component<any, PostState> {
           >
             {I18NextService.i18n.t("top")}
           </label>
+          {enableDownvotes(this.state.siteRes) && (
```

Comment:
Remove this enableDownvotes check, as well as the `hideControversial` props to the other two.

There are a lot of sorts that don't make sense when votes are disabled, not just controversial, so that should be done in a separate PR.